andela/hc-wits-kla
# #158175324 Add writing and sharing of blog posts.

#### What does this PR do?
This PR allows allows a user to write blog posts and also visitors to read and comment on published posts.
#### Description of Task to be completed?
This tasks involves creating a basic blog that allows logged in users to create categories, write blog posts and share blog posts.
#### How should this be manually tested?
- Clone the application and install the requirements in requirements.txt `pip install -r requirements.txt`
- Run migrations and then start the application `./manage.py migrate blog` and then `python manage.py runserver`
- You may also need to compress the static files `./manage.py compress`
- Login and navigate to the my posts tab
- Click Add Category button and add a category.
- Click Add post button
- Add posts detail publishing date and time should be today.
- Navigate to the blog tab and view post, click the link to view details and make a comment on it.
#### What are the relevant pivotal tracker stories?
[#158175324](https://www.pivotaltracker.com/story/show/158175324)
#### Screenshots 
<img width="1440" alt="screen shot 2018-07-05 at 14 45 13" src="https://user-images.githubusercontent.com/23418080/42323394-c135360a-8068-11e8-9139-3c838fdf3578.png">
<img width="1440" alt="screen shot 2018-07-05 at 14 45 24" src="https://user-images.githubusercontent.com/23418080/42323403-c766b684-8068-11e8-815c-d7aa5e9805e0.png">
<img width="1440" alt="screen shot 2018-07-05 at 15 43 51" src="https://user-images.githubusercontent.com/23418080/42323896-51152fe0-806a-11e8-9bcf-3e22893fa855.png">

<img width="1439" alt="screen shot 2018-07-05 at 14 44 38" src="https://user-images.githubusercontent.com/23418080/42323445-e271c252-8068-11e8-8f75-dab4e1a39a5f.png">
<img width="1436" alt="screen shot 2018-07-05 at 14 44 16" src="https://user-images.githubusercontent.com/23418080/42323453-e772db92-8068-11e8-910e-96e4f3ae7b83.png">
